robot/EDA/resources/TriggerHandlersPageObject.py

```diff
@@ -0,0 +1,195 @@
+import time
+
+from selenium.webdriver.common.keys import Keys
+from cumulusci.robotframework.pageobjects import HomePage
+from cumulusci.robotframework.pageobjects import BasePage
+from cumulusci.robotframework.pageobjects import DetailPage
+
+from cumulusci.robotframework.pageobjects import pageobject
+from locators import trigger_handlers_locators
+
+
+
+@pageobject("Home", "TriggerHandlers")
+class TriggerHandlersHomePage(BasePage):
```

Comment:
@dashattuck Sorry I missed your messages this weekend -- with the EDA 1.84 fix, I don't think we need to interact with the Trigger Handlers tab and records with the Preferred Phone functionality anymore. Turning the feature on/off is now controlled entirely on the EDA Settings page.

Reply:
already removed.  Thanks Bobby!